config/settings.yml

```diff
@@ -14,3 +14,6 @@ plugins:
     default: ''
   patreon_creator_refresh_token:
     default: ''
+  patreon_declined_pledges_grace_period_days:
+    default: 30
```

Comment:
Hey there! Quick introduction, my name's Stephen, and I work on the payments team at Patreon.

I think that the 30 default might be too generous. The vast majority of decline fixes (I don't have the raw data on hand) are going to be resolved within 1 week. I think it's safe for creators to make the assumption that after a week, almost all of the patrons who haven't yet fixed their payment simply won't, and they might be better served by dropping access for those people, rather than prolonging it for 3 more weeks.

Reply:
Thank you @alts for the details. Okay I will update the default value as 7 days :+1: